app/javascript/mastodon/locales/ar.json

```diff
@@ -164,4 +164,4 @@
   "video_player.toggle_sound": "تبديل الصوت",
   "video_player.toggle_visible": "إظهار / إخفاء الفيديو",
   "video_player.video_error": "تعذر تشغيل الفيديو"
-}
\ No newline at end of file
+}
```

Comment:
Please only delete the last newline. } Does not have to be deleted. Please make sure that there is no difference in the last line.